src/pkg/cli/client/byoc/baseclient.go

```diff
@@ -142,7 +142,12 @@ func (b *ByocBaseClient) RemoteProjectName(ctx context.Context) (string, error)
 		return "", errors.New("no projects found")
 	}
 	if len(projectNames) > 1 {
-		return "", fmt.Errorf("multiple projects found; use the --project-name flag to specify a project: %v", projectNames)
+		var projectList = ""
```

Comment:
In this case, it's probably cleaner to create a custom error `type ErrMultipleProjects struct { ProjectNames []string }` with an `Error() string` method.